data_demos/NF_hackathon_exomeseq_datademo.Rmd

```diff
@@ -0,0 +1,216 @@
+
+We tested the hypothesis whether the number of mutations in a set of genes can be used for clustering the data/samples into meaningful groups. For this, we first created a scoring matrix by converting variant data into numerical variables for PCA. 
+
+```{r r making a scoring matrix, eval=TRUE, results='hide', message=FALSE, warning=FALSE}
+
+# Filter select columns for analysis
+select_df <- exome_data[, c("Hugo_Symbol", "Entrez_Gene_Id", "Chromosome", "Start_Position", "Variant_Type", 
+                            "Transcript_ID", "HGVSp_Short", "BIOTYPE", "SWISSPROT", "RefSeq", "PolyPhen", 
+                            "IMPACT", "FILTER", "id", "assay", "diagnosis", "individualID", "isCellLine", 
+                            "organ", "sex", "species", "study", "tumorType", "gnomAD_AF", "gnomAD_AFR_AF",
+                            "gnomAD_AMR_AF", "gnomAD_ASJ_AF", "gnomAD_EAS_AF", "gnomAD_FIN_AF", "gnomAD_NFE_AF",
+                            "gnomAD_OTH_AF", "gnomAD_SAS_AF")]
+
+# Make dummy variables for PCA
+select_df <- select_df %>% mutate(isSNP = as.numeric(select_df$Variant_Type == "SNP")) 
+select_df <- select_df %>% mutate(isDEL = as.numeric(select_df$Variant_Type == "DEL")) 
+select_df <- select_df %>% mutate(isDNP = as.numeric(select_df$Variant_Type == "DNP"))  
+select_df <- select_df %>% mutate(isINS = as.numeric(select_df$Variant_Type == "INS")) 
+select_df <- select_df %>% mutate(isONP = as.numeric(select_df$Variant_Type == "ONP"))  
+select_df <- select_df %>% mutate(isTNP = as.numeric(select_df$Variant_Type == "TNP")) 
+select_df <- select_df %>% mutate(totHits = rowSums(select_df[,c(33:38)]))
+
+# Filter out well known common variants to look for rare mutations, and select mutations with annotated "HIGH" impact on proteins
+impact_df <- select_df[(select_df$IMPACT == "HIGH" & select_df$FILTER != "common_variant"),]
+```
+
+We used the number of mutational events (_hits_) to the genes as a scoring metric for each gene to do our principal component analysis. Since some genomic variants can be quite common and many of them are well tolerated, we filtered out all known common_variants in the genes and selected for the ones with high impact on proteins to strengthen the signal for PCA. 
+
+We notice that the PCA seems to identify two different clusters in the data based on this metric, and also identifies one sample as an outlier. However, even with our efforts to select relevant mutations, the first principal component explains only 13% of the variance in the data. We encourage the participants to explore other variables present in the dataset and combinations thereof to find more significant sources of variance within the data. The later panels show the different types of mutations observed in some of the genes that constitute the first and second principal components of the analysis.
```

Comment:
suggest create a new chunk for the PC loading summary plots, with the description after the first PCA plot